watertap/examples/case_studies/municipal_treatment/tests/test_municipal_treatment.py

```diff
@@ -0,0 +1,128 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+
+import pytest
+from io import StringIO
+from pyomo.environ import value
+from watertap.examples.case_studies.municipal_treatment.municipal_treatment import main
+from idaes.core.util import get_solver
+
+# -----------------------------------------------------------------------------
+@pytest.mark.component
+def test_municipal_treatment():
+    m = main()
+
+    report_io = StringIO()
+    m.fs.feed.report(ostream=report_io)
```

Comment:
I don't think you should be testing the string outputs here - these are rather fragile and we only do them in the unit models tests to make sure the method is functional. At the flowsheet level, you would be much better off just testing the expected value of the key variables with `pytest.approx`.

Reply:
I got rid of the report comparisons.